test/Dialect/BaseModelica/Transforms/CallCSE/idempotent-configurations.mlir

```diff
@@ -92,4 +92,60 @@ module @InductionVariables {
             bmodelica.equation_instance %t1 {indices = #modeling<multidim_range [0, 1]>} : !bmodelica.equation
         }
     }
+}
+
+// -----
+
+module @ArrayResult {
```

Comment:
No need to have a named module.

Reply:
Did it to "document" what the test is for, as the file contains multiple. Added a CHECK-LABEL to it now to make it make a bit more sense. Take a look at the whole file for context

Reply:
Ok, I did miss it in the other modules. Looks good.